src/components/manifold-plan-table/manifold-plan-table.tsx

```diff
@@ -227,7 +229,7 @@ export class ManifoldPlanTable {
           planId,
         },
       })
-      .then(() => {
+      .finally(() => {
```

Comment:
This is unrelated, but the redirect was failing for me because ublock blocks Segment